aerogear/voyager-server
# AEROGEAR-8560 - compilation check in release validity check

https://issues.jboss.org/browse/AEROGEAR-8560

Verification:
1. Run `npm run compile`
2. Run `TAG=0.4.0 ./scripts/validateRelease.sh.` All should be good
3. Create some compilation error somewhere
4. Run `npm run compile`
5. Run `TAG=0.4.0 ./scripts/validateRelease.sh.` Exit code should be 1
